### PR TITLE
Rubocop cleanup

### PR DESCRIPTION
**Pull request by harrisj**
Since we change the rubocop config, I thought it would be good to clean up some simple violations across the code (no major changes like renaming functions)

### CODE REVIEW

File: app/controllers/admin/auctions_controller.rb [hunk 1]
```diff
@@ -66,7 +66,7 @@ def update
         format.html { redirect_to "/admin/auctions" }
         format.json do
           render json: auction, serializer: Admin::AuctionSerializer
-exit        end
```
Comment (harrisj) [inline app/controllers/admin/auctions_controller.rb]: Not sure what happened here

Comment (jessieay) [inline app/controllers/admin/auctions_controller.rb]: this is the best typo I've ever seen

File: .rubocop.yml [hunk 2]
```diff
@@ -102,6 +102,10 @@ Style/DotPosition:
   - leading
   - trailing
 
+Style/IfUnlessModifier:
```
Comment (jessieay) [inline .rubocop.yml]: I think I agree with turning this off if this is it: https://github.com/bbatsov/ruby-style-guide#if-as-a-modifier

but what does this have to do with max line length?

File: spec/requests/auctions_spec.rb [hunk 3]
```diff
@@ -0,0 +1,334 @@
+require 'rails_helper'
+
+RSpec.describe AuctionsController do
```
Comment (jessieay) [inline spec/requests/auctions_spec.rb]: I think this file was accidentally re-added (split into two request specs in a recent PR, this one should be deleted)

File: spec/requests/bids_spec.rb [hunk 4]
```diff
@@ -110,7 +110,7 @@
       let(:api_key) { FakeGitHub::VALID_API_KEY }
       let(:auction) { FactoryGirl.create(:auction, :running) }
       let(:current_auction_price) do
-        auction.bids.sort_by {|b| b.amount}.first.amount
+        auction.bids.sort_by(&:amount).first.amount
```
Comment (jessieay) [inline spec/requests/bids_spec.rb]: YAY

File: spec/services/losing_bidder_email_sender_spec.rb [hunk 5]
```diff
@@ -10,12 +10,12 @@
...
           second_losing_bid = create(:bid, amount: 101, auction: auction)
           mailer_double = double(deliver_later: true)
-          allow(AuctionMailer).to receive(:losing_bidder_notification).
-            with(losing_bid).
-            and_return(mailer_double)
-          allow(AuctionMailer).to receive(:losing_bidder_notification).
-            with(second_losing_bid).
-            and_return(mailer_double)
+          allow(AuctionMailer).to receive(:losing_bidder_notification)
+            .with(losing_bid)
```
Comment (jessieay) [inline spec/services/losing_bidder_email_sender_spec.rb]: oh interesting I usually prefer the `.` at the ends of lines --- but whatever works!

**Comment by jessieay** [comment]
@harrisj this needs a rebase. should be passing CI once that happens, too!

File: .rubocop.yml (hunk 2 above)
Comment (harrisj) [inline .rubocop.yml]: Oh, this isn't anything to do with max-line-length, I just wanted to let you do your multiline if blocks without rubocop complaining

File: spec/requests/auctions_spec.rb (hunk 3 above)
Comment (harrisj) [inline spec/requests/auctions_spec.rb]: Got it. Will delete

File: spec/services/losing_bidder_email_sender_spec.rb (hunk 5 above)
Comment (harrisj) [inline spec/services/losing_bidder_email_sender_spec.rb]: We keep going back and forth on this one (I do too, but I know some other devs did not and I didn't feel strongly either way myself)

Comment (jessieay) [inline spec/services/losing_bidder_email_sender_spec.rb]: same same!

**Comment by jessieay** [comment]
Looks amazing! Thanks for doing all of this cleanup 🚜
